docs/usage/project/namespaced-cloud-profiles.md

```diff
@@ -0,0 +1,60 @@
+---
+title: NamespacedCloudProfiles
+---
+
+# `NamespacedCloudProfile`s
+
+`NamespacedCloudProfile`s are resources in Gardener that allow project-level customization of `CloudProfile`s.
+They enable project administrators to create and manage cloud profiles specific to their projects and reduce the operational burden on central Gardener operators.
+These profiles inherit from a parent `CloudProfile` and can override or extend certain fields while maintaining backward compatibility.
+
+Project viewers have the permission to see `NamespacedCloudProfile`s associated with a particular project.
+Project members can generally create, edit, or delete `NamespacedCloudProfile`s but with some exceptions (see the [restrictions](#field-modification-restrictions) outlined below).
+
+When creating or updating a `Shoot`, the cloud profile reference can be set to point to a `NamespacedCloudProfile`, allowing for more granular and project-specific configurations.
+The modification of a `Shoot`'s cloud profile reference is restricted to switching from a `CloudProfile` to a descendant `NamespacedCloudProfile`.
+Changing the reference from one `NamespacedCloudProfile` to another `NamespacedCloudProfile` or even to another `CloudProfile` is not allowed.
```

Comment:
Interesting, I was not aware of it. So, when a `Shoot` switches from a `CloudProfile` to a `NamespacedCloudProfile` (maybe because of a temporary expiration date extension), I cannot switch back eventually to the global `CloudProfile`?

Reply:
Exactly, currently this is prevented by the `Shoot` core validation. [IIRC](https://github.com/gardener/gardener/pull/10093#issuecomment-2239053593) the possibility to switch to the parent `CloudProfile` has been intentionally restricted to keep the validations required simple for now.

Reply:
OK. I guess this is something that should be addressed, but probably in a new/subsequent PR. I guess, a standard use-case is exactly what I described above: Temporary extension of an expiration date, but eventually folks might want to switch back again and cleanup the temporary `NamespacedCloudProfile` resources.

Reply:
Agreed. I will create a followup PR immediately once this one is merged. 👍